src/prx/user.py

```diff
@@ -15,7 +16,38 @@ def parse_prx_csv_file(prx_file: Path):
     return pd.read_csv(prx_file, comment="#"), parse_prx_csv_file_metadata(prx_file)
 
 
+def spp_vt_lsq(df, p_ecef_m):
+    df = df[df.D_obs_hz.notna()].reset_index(drop=True)
+    df["D_obs_mps"] = df.D_obs_hz * cGpsSpeedOfLight_mps / df.carrier_frequency_hz
```

Comment:
According to https://gnss-sdr.org/docs/sp-blocks/observables/#doppler-shift-measurement, the conversion to mps involves a minus sign.

`df["D_obs_mps"] = df.D_obs_hz * cGpsSpeedOfLight_mps / df.carrier_frequency_hz`

Then, the corrected pseudorange rate would have the following signs:
`df["D_obs_corrected_mps"] = (
        df.D_obs_mps.to_numpy().reshape(-1, 1)
        + df.dclock_mps.to_numpy().reshape(-1, 1)
        - df["satellite_los_velocities"].to_numpy().reshape(-1, 1)
    )`

All those sign errors are unobservable for fixed data collect.

We should probably put a short dynamic trajectory and verify if the estimated velocity using the doppler observations corresponds to the time difference of the estimated position.

Reply:
Totally agree, added https://github.com/jtec/prx/issues/66

Reply:
Thanks for checking the sign, made the change.